imports/server/server-render.js

```diff
@@ -5,6 +5,13 @@ onPageLoad((sink) => {
     '  <meta charset="utf-8">\n' +
     '  <meta http-equiv="X-UA-Compatible" content="IE=edge">\n' +
     '  <meta name="viewport" content="width=device-width, initial-scale=1">\n' +
+    '  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">\n' +
+    '  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">\n' +
+    '  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">\n' +
+    '  <link rel="manifest" href="/site.webmanifest">\n' +
+    '  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">\n' +
+    '  <meta name="msapplication-TileColor" content="#000000">\n' +
+    '  <meta name="theme-color" content="#ffffff">' +
```

Comment:
mind adding the `\n` at the end of this one too?

Reply:
d'oh